src/App.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import './App.css';
+import data from './data/resume.json';
+
+function BuildBasicsElement(basics) {
+  /*
+    todo: Elements in basic info remaining below.
+    For profiles, would have considered creating a for loop
+    to get through each profile.
+    "profiles": [{
+      "network": "Twitter",
+      "username": "john",
+      "url": "http://twitter.com/john"
+    }]
+  */
+  const { location } = basics;
+  return (
+    <div className="App" align="left">
+      <h1>{basics.name}</h1>
+      <h3>{basics.label}</h3>
+      <h5>
+        {basics.location.address} {location.postalCode} {location.city} {location.region} {location.country}
+        <br></br>
+        {basics.phone}
+        <br></br>
+        {basics.email}
+        <br></br>
```

Comment:
<br> works, but it's better to use padding-bottom. that way, you aren't expressing style as structure.
i.e. HTML = structure, not style. but you're using <br> as styling :-)
Have you seen the <p> tag?